<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HalamanUtama</class>
 <widget class="QWidget" name="HalamanUtama">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QTableView" name="tableView_buku">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>611</width>
     <height>461</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>430</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Judul            :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>470</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Pengarang    :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>510</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tahun            :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>550</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Kategori        :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_judul">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>430</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit__pengarang">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>470</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_tahun">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>510</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="button_tambahBuku">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>680</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ubah">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>680</y>
     <width>85</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Update</string>
   </property>
  </widget>
  <widget class="QPushButton" name="button_hapusBuku">
   <property name="geometry">
    <rect>
     <x>870</x>
     <y>680</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hapus</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>760</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Status :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_status">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>760</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_kategori">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>550</y>
     <width>281</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>401</width>
     <height>351</height>
    </rect>
   </property>
   <property name="columnCount">
    <number>5</number>
   </property>
   <column>
    <property name="text">
     <string>judul</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>pengarang</string>
    </property>
   </column>
   <column/>
   <column/>
   <column/>
  </widget>
  <widget class="QLabel" name="label_tampilan">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>370</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_informasi">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>310</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Informasi</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>0</y>
     <width>1920</width>
     <height>1080</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/gambar/gambar/Untitled-2.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>tableView_buku</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_4</zorder>
  <zorder>lineEdit_judul</zorder>
  <zorder>lineEdit__pengarang</zorder>
  <zorder>lineEdit_tahun</zorder>
  <zorder>button_tambahBuku</zorder>
  <zorder>pushButton_ubah</zorder>
  <zorder>button_hapusBuku</zorder>
  <zorder>label_5</zorder>
  <zorder>label_status</zorder>
  <zorder>lineEdit_kategori</zorder>
  <zorder>tableWidget</zorder>
  <zorder>label_tampilan</zorder>
  <zorder>pushButton_informasi</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
